week-1/Homework/mandatory/6-playing-computer.js

```diff
@@ -20,11 +20,12 @@ let a = 6;
 
 const f1 = function(a, b) {
   return a + b;
-};
+}
 
 const f2 = function(a, b) {
+  let x = 10;
   return a + b + x;
-};
+}
```

Comment:
Did you get stuck doing this exercise? It's been a few weeks since then, maybe you want to give it another try ;)